website/client/components/static/faq.vue

```diff
@@ -1,18 +1,47 @@
 <template lang="pug">
-  .container-fluid
+  .container-fluid(role="tablist")
     .row
       .col-12.col-md-6.offset-md-3
-        h1 {{ $t('frequentlyAskedQuestions') }}
-        .faq-question(v-for='(heading, index) in headings')
-          h2.accordion(@click='setActivePage(heading)') {{ $t(`faqQuestion${index}`) }}
-          div(v-if='pageState[heading]', v-markdown="$t('webFaqAnswer' + index, replacements)")
+        h1#faq-heading {{ $t('frequentlyAskedQuestions') }}
+        b-card.faq-question(no-body, v-for='(heading, index) in headings', :key="index")
```

Comment:
is there any reason to use `b-card`/`b-card-body` instead of a `div` I didn't see any differences with `div`